Make.config

```diff
@@ -112,7 +112,7 @@ WATCHOS_NUGET_VERSION_FULL=$(WATCHOS_NUGET_VERSION_NO_METADATA)+$(NUGET_BUILD_ME
 
 # Xcode version should have both a major and a minor version (even if the minor version is 0)
 XCODE_VERSION=12.1
-XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_12.1_GM_seed.xip
+XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_12.1.xip
 XCODE_DEVELOPER_ROOT=/Applications/Xcode_12.1.0-GM.app/Contents/Developer
```

Comment:
^ don't you want/need this updated for the device bots ?

Reply:
@spouliot the device bots use provisioning, the provisionator file gets updated by just changing this Make.config. I use a Make file to generate the csx file based on the configuration in the Make.config :)